book/src/miden_node_setup_tutorial.md

```diff
@@ -0,0 +1,74 @@
+# Miden Node Setup Tutorial
+
+To run the Miden tutorial examples, you will need to connect to a running Miden node.
+
+There are two ways to connect to a Miden node:
+1) Run the Miden node locally
+2) Connect to the Miden testnet
+
+## Running the Miden Node Locally
+
```

Comment:
Can you add a prerequisite section here and say, to run the node, you need to 

- download the node package
- a genesis.toml [say what it is]
- a miden-node.toml [say what it is]

but we will support the two setup files via our repo